### PR TITLE
types

### PR DESCRIPTION
**Pull request by TyPlox**
I added types and fixed a little bug in Search.activity

### CODE REVIEW

File: lib/search.js [hunk 1]
```diff
@@ -150,7 +150,7 @@ class Search {
      * @since 1.7.0
      */
     activity(activityID=null, filter=null, page=1, amount=5) {
-        if (activityID && typeof activityID !== "string") { throw new Error("activityID is not a number."); }
+        if (activityID && typeof activityID !== "number") { throw new Error("activityID is not a number."); }
```
Comment (Butterstroke) [inline lib/search.js:153]: Oh! Nice catch. 👍

File: package.json [hunk 2]
```diff
@@ -31,7 +39,8 @@
   },
   "license": "MIT",
   "dependencies": {
-    "node-fetch": "^2.6.1"
+    "node-fetch": "^2.6.1",
+    "typescript": "^4.2.4"
```
Comment (Butterstroke) [inline package.json:43]: Please list typescript to a dev dependency. We don't need to install typescript as a regular dependency to get the type definitions for installs.

**Review by Butterstroke — CHANGES_REQUESTED** [review]
This looks good. Just change Typescript as a dev dependency and I'll approve of it for the next release.

File: package.json (hunk 2 above)
Comment (TyPlox) [reply package.json]: Change done

**Review by Butterstroke — APPROVED** [review]

**Comment by Butterstroke** [comment]
Thanks for the work! 😄 You should see an update with these changes in the following days.

**Pull request merged**